Source/WebCore/platform/graphics/FontPlatformData.h

```diff
@@ -301,7 +301,7 @@ class FontPlatformData {
     CTFontRef ctFont() const;
 #endif
 
-#if PLATFORM(WIN) || PLATFORM(COCOA)
+#if PLATFORM(COCOA)
     bool isSystemFont() const { return m_isSystemFont; }
```

Comment:
Should this just be moved into the `USE(CORE_TEXT)` block above?

Reply:
`isSystemFont()` is used from `PLATFORM(COCOA)` code block too.
https://github.com/WebKit/WebKit/blob/535d9cfa7e4a6c1cce863562cf29272717bdb1b1/Source/WebCore/rendering/RenderBlock.cpp#L3537